app/components/annotation/Article.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
+import { addHighlight } from 'actions/actions';
+import { connect } from 'react-redux';
 
-export default React.createClass({
+const mapDispatchToProps = dispatch => {
```

Comment:
exposes the `dispatch` function to props inside this component - it's `react-redux`'s design pattern

https://github.com/reactjs/react-redux